common/src/mailbox_api.rs

```diff
@@ -276,6 +277,36 @@ impl Default for InvokeDpeReq {
     }
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub data_size: u32,
+    pub data: [u8; ExtendPcrReq::DATA_MAX_SIZE],
+    _pad: [u8; 4 - (ExtendPcrReq::DATA_MAX_SIZE % 4)],
+}
+// No command-specific output args
+impl ExtendPcrReq {
+    pub const DATA_MAX_SIZE: usize = sha384::SHA384_BLOCK_BYTE_SIZE - sha384::SHA384_HASH_SIZE - 1;
```

Comment:
It seems like this leaks HW details that would be nicer to hide. Is there a need to allow the data size to exceed 48 bytes? Alternatively, could we try rejiggering the PCR driver to handle multiple blocks? It's not clear to me that the current limitation is HW-based.

Reply:
I think there is a need for the `data_size` attribute to exceed 48 bytes, since `data` is used as an input for the method `PcrBank::extend_pcr`, which can process more than 48 bytes at once in the internal hash digest. To me it looks like this function can take up to `128 - 1 - 48 = 79` bytes as payload (as we already discussed earlier) or am I confusing things here?

Reply:
So you propose, that we make the `data_size` and `data` in `ExtendPcrReq` entirely variable (wihtout nay `DATA_MAX_SIZE`) and then handle multiple blocks in the `PcrBank::extend_pcr` method?

Reply:
More or less. I think we still need some maximum size, but 79 bytes seems fairly arbitrary. Either 48 or 96 or 192 bytes seems a bit cleaner from an API perspective.

Reply:
I see you point, I adjusted the command in such a way, that it should now be possible to pass a variable amount of data, currently limited to 192 bytes as you proposed.

Reply:
Any thoughts on this? @bluegate010

Reply:
Sorry for the delay! Will resolve this and add further comments inline.